dags/lib/config_operators.py

```diff
@@ -0,0 +1,36 @@
+# This is meant to be used in DAGS depending on the KubernetesPodOperator.
+# It may not be used in older dags, but please use it in new ones.
```

Comment:
This comment originally refers to the kube_config_etl object (here at line 9). Was it intentional to place it at the top of the module here?

Reply:
Oups! Corrigé